src/NUnitCommon/nunit.agent.core.tests/nunit.agent.core.tests.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net462;net8.0</TargetFrameworks>
+    <ExposedInternals>true</ExposedInternals>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)'=='net462' OR '$(TargetFramework)'=='net6.0' OR '$(TargetFramework)'=='net8.0'">
```

Comment:
There is no `net6.0` target anymore.

Reply:
Yup. I'll make that change but wait to see what else you have. You may also notice that there are untested agent-core targets: netcoreap3.1 and net6.0. I keep them because I know I'll use them later for separate user-installable agents. I see no point of doing it for anything older than those.

Reply:
I left it in for now as it does no harm.